packages/pro/tree/src/types.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://github.com/IDuxFE/idux/blob/main/LICENSE
+ */
+
+import type { ExtractInnerPropTypes, ExtractPublicPropTypes, MaybeArray, VKey } from '@idux/cdk/utils'
+import type { EmptyProps } from '@idux/components/empty'
+import type { HeaderProps } from '@idux/components/header'
+import type {
+  TreeCheckStrategy,
+  TreeCustomAdditional,
+  TreeDragDropOptions,
+  TreeNode,
+  TreeNodeDisabled,
+} from '@idux/components/tree'
+import type { DefineComponent, HTMLAttributes, PropType } from 'vue'
+
+export const proTreeProps = {
+  checkedKeys: { type: Array as PropType<VKey[]>, default: undefined },
+  expandedKeys: { type: Array as PropType<VKey[]>, default: undefined },
+  loadedKeys: { type: Array as PropType<VKey[]>, default: undefined },
+  selectedKeys: { type: Array as PropType<VKey[]>, default: undefined },
+
+  checkable: { type: Boolean, default: false },
+  childrenKey: { type: String, default: undefined },
+  checkStrategy: { type: String as PropType<TreeCheckStrategy>, default: 'all' },
```

Comment:
```suggestion
  cascaderStrategy: { type: String as PropType<CascaderStrategy>, default: 'all' },
```
改成这个 API 吧，tree 那边也同步改下？注意下兼容之前的 cascade + checkStrategy API。